<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Label_pic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_ip">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_port">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_UDP">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_serial">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_status">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>15</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_action">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>action</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasname">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>108</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>UAS_NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>138</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>UAS_ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_uasname">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>104</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_uasid">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>134</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_2">
    <property name="geometry">
     <rect>
      <x>869</x>
      <y>137</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>SYS_ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_sysid">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>133</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_compoid">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>166</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPON_ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>170</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UAS_TIME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_uastime">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>166</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>204</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD_TYPE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_loadtype">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_6">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>204</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS_STATUS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_gpsstatus">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_planetype">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>230</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_7">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>234</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PLANE_TYPE</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasname_2">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>300</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>飞行里程:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_8">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>394</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电台固件:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_9">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>362</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电台序列:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_10">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>425</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>飞控序列:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_11">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>330</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起降次数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_uasid_12">
    <property name="geometry">
     <rect>
      <x>702</x>
      <y>454</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>飞控固件:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info1">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>300</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info2">
    <property name="geometry">
     <rect>
      <x>763</x>
      <y>330</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info3">
    <property name="geometry">
     <rect>
      <x>763</x>
      <y>362</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info4">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>394</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info5">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>424</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info6">
    <property name="geometry">
     <rect>
      <x>765</x>
      <y>454</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_other">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_info7">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>493</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
